app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardScreen.kt

```diff
@@ -362,8 +364,19 @@ fun KeyboardScreen(
                     Row {
                         row.forEachIndexed { j, key ->
                             Column {
+                                val ghostKey =
```

Comment:
The ghost keys should probably be defined in the keyboard definitions themselves, but this is much simpler.

Reply:
Hmm, yeah, maybe. I assume by this you mean a `KeyItemC` should have an optional reference to another `KeyItemC`, rather than a `KeyItemC` having, for example, a `ghostSwipes` entry (analogous to `swipes`)?
I would think it's kind of strange to have a `ghostSwipes` entry, since then you could put any key down. It feels to me like, if you wanted to do that, you should just define them in `swipes`, rather than using ghost keys to override the numeric key in the same position.

Reply:
For sure, an additional layer to swipes, or another map. No changes needed for this tho, its good.